concepts/05 UI Components/Form/01 Getting Started with Form/00 Getting Started with Form.md

```diff
@@ -0,0 +1,9 @@
+The Form component creates data entry UI for the underlying data object. The interface consists of automatically-arranged label-editor pairs that correspond to data fields. You can change data values, validate user input, and send resulting data to the server.
+
+This tutorial shows you the basics of **Form** component configuration. The preview below shows the expected result:
```

Comment:
"shows you the basics" means what? What does it show? A tutorial usually explains something, gives information, or describes a specific functionality - images show or illustrate things. You also use **shows** in both sentences.

Reply:
Got it from Vladimir Abadzhev, he modified our template for 'getting started' tutorials. My initial option was something like

'This tutorial shows how to configure basic Form features. As a result, you will get a UI component that looks as follows:'

Please also take into account that the code in the line below displays a special frame with a working application:
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/33040934/106899898-f69ed080-6706-11eb-8a6f-8583992a8587.png)

Reply:
This tutorial explains how to configure the **Form** component.
OR
This tutorial shows basic **Form** component configuration.

(in my opinion, I would leave out "The preview below shows the expected result" because the first sentence already states what the aim of this topic is)